### ISSUE TITLE
Missing optional with jsonb

### ISSUE THREAD

**Issue by stuarthendren** [vojtechhabarta/typescript-generator#491]
When using the following configuration properties

```
<jsonLibrary>jsonb</jsonLibrary>
<optionalProperties>useSpecifiedAnnotations</optionalProperties>
<optionalAnnotations>
  <annotation>javax.annotation.Nullable</annotation>
</optionalAnnotations>
<optionalPropertiesDeclaration>questionMark</optionalPropertiesDeclaration>
```

The Nullable annotation seems to be ignored and all properties some out without the question mark. Moving the annotation from getter, setter, or field does not seem to affect this either. We do get the question mark in projects using jackson, so possible down to the jsonb.

**Comment by vojtechhabarta** [comment]
@rmannibucau already implemented this and it is merged in master.
I will let you know when it is released.

**Comment by vojtechhabarta** [comment]
I forgot about this issue, it was released in [v2.24.612](https://github.com/vojtechhabarta/typescript-generator/releases/tag/v2.24.612), current release is [v2.25.695](https://github.com/vojtechhabarta/typescript-generator/releases/tag/v2.25.695).
